### PR TITLE
feat_BGDIINF_SB-2666: adding a feedback and link to CMS buttons

### PR DESCRIPTION
**Pull request by procrastinatio**
See https://jira.swisstopo.ch/browse/BGDIINF_SB-2666 and  https://jira.swisstopo.ch/browse/BGDIINF_SB-2674

Please translate the strings, `app_feedback_*` and `cms_link_*`  accordingly.

[Test link](https://sys-map.dev.bgdi.ch/preview/feat_bgdiinf_sb-2666_link_to_cms/index.html)

### CODE REVIEW

**Comment by pakb** [comment]
there's also an issue with the "Feedback" button size on mobile
image
you might need to remove some fixed width somewhere

File: setup-fontawesome.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import { library } from '@fortawesome/fontawesome-svg-core'
```
[the remark this answers is not in the document]
Comment (procrastinatio) [reply setup-fontawesome.js:1]: Well, you did added with #95fed2b53df3adf2e0a0ee6d7d6bdc6c4d948988

[the remark this answers is not in the document]
Comment (pakb) [reply setup-fontawesome.js:1]: yes but here it's a copy outside of `src/` that you are adding 😉

File: src/modules/menu/components/menu/LinkButton.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import { mapGetters, mapState } from 'vuex'
+
+export default {
+    computed: {
+        ...mapGetters(['isDesktopMode']),
+        ...mapState({
+            currentLang: (state) => state.i18n.lang,
+        }),
+        cmsUrl() {
+            return "https://www.geo.admin.ch/" + this.currentLang + "/map-viewer/map-viewer.html"
```
[the remark this answers is not in the document]
Comment (procrastinatio) [reply src/modules/menu/components/menu/LinkButton.vue:29]: OK. But easier to limit the length of the lines. I'm still  traumatized by the linters.

File: src/modules/menu/components/menu/MenuFeedback.vue [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+<script>
+import LangSwitchToolbar from '@/modules/menu/components/FeedbackToolbar.vue'
+
+export default {
+    components: {
+        FeedbackToolbar,
+    },
+}
+</script>
```
[the remark this answers is not in the document]
Comment (procrastinatio) [reply src/modules/menu/components/menu/MenuFeedback.vue]: OK. deleted

**Comment by pakb** [comment]
you can use the key `test_map_give_feedback` for the "Give feedback" button and `cms_link_button_title` for the "Information" button

**Comment by procrastinatio** [comment]
OK. translation strings changed et repushed.

**Review by pakb — APPROVED** [review]
👍

**Pull request merged**